[16.0][ADD] account_invoice_reimport_from_attachment (#2241)
OCA/account-invoicing: account_invoice_reimport_from_attachment/readme/USAGE.md

## Usage

1. Open a draft invoice.

> 2. Click **Re-import from Attachment**.

Reviewer: ```suggestion
2. Click **Reimport from Attachment**.
```

3. Select the attachment to import from (among the invoice attachments).
4. Confirm.

### Notes

- The action is only available on **draft** invoices.

> - The re-import **deletes current invoice lines** and recreates them from the selected attachment.

Reviewer: ```suggestion
- The reimport **deletes the current invoice ** and recreates a new one from the selected attachment.
```

> - Access is restricted to users in the **Invoice Re-import from Attachment** group.

Reviewer: ```suggestion
- Access is restricted to users in the **Invoice Reimport from Attachment** group.
```
